frontend/templates/views/partials/format-filter-hint.j2

```diff
@@ -3,14 +3,16 @@
 
 {% do doc.amp_dependencies.add('amp-user-notification', '0.1') %}
 <amp-user-notification class="ap-m-format-filter-hint" id="format-filter-hint" layout="nodisplay">
-  <div class="ap-m-format-filter-hint-content">
-    <div class="ap-m-format-filter-hint-note">{{ _('Select your format for a more streamlined experience') }}</div>
+</amp-user-notification>
 
```

Comment:
Please add a comment that the next element needs to directly follow the user notification